mage_ai/__init__.py

```diff
@@ -60,11 +61,13 @@ def connect_data(df, name):
     return feature_set
 
 
-def clean(df, pipeline_uuid=None, pipeline_path=None):
+def clean(df, pipeline_uuid=None, pipeline_path=None, remote_pipeline_uuid=None):
```

Comment:
when calling clean method with remote_pipeline_uuid, we need to pass in api_key as well. it'll be used independently in production environment without launching server.

Reply:
I see